inginious/frontend/plugins/contact_page/pages/api/contact_page.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of UNCode. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+
+""" Contact page """
+
+import web
+import requests
+import json
+from inginious.frontend.pages.utils import INGIniousPage
+from inginious.frontend.plugins.contact_page.pages.api.slack_url_error import SlackURLError
+
+subject_text = {
+    "subject-comment": "Comentario o reporte de un problema",
+    "subject-new-course": "Peticion para crear un nuevo curso",
+}
+URL_channel = {
+    "subject-comment": "",
+    "subject-new-course": "",
+}
+subject_new_course_id = "subject-new-course"
+
+base_renderer_path = "frontend/plugins/contact_page/pages/templates"
+
+
+def set_url_channel(main_message_channel, new_course_channel):
+    """ Define the URL directions where do the request """
+    global URL_channel
+    if main_message_channel != "":
+        URL_channel["subject-comment"] = main_message_channel
+        if new_course_channel != "":
+            URL_channel["subject-new-course"] = new_course_channel
+        else:
+            URL_channel["subject-new-course"] = main_message_channel
+    else:
+        raise SlackURLError("The main slack's URL is empty")
+
+
+def create_payload(data):
+    """ Create the payload to be send """
+    blocks = [create_slack_text_block("Asunto", subject_text[data["subject_id"]]),
+              create_slack_text_block("Email", data["email"]),
+              create_slack_text_block("Nombre", data["name"])]
+    if is_new_course_request(data["subject_id"]):
+        blocks.append(create_slack_text_block("Nombre del curso", data["courseName"]))
+    blocks.append(create_slack_text_block("Comentario", data["textarea"]))
```

Comment:
Let's name the block's title in English as UNCode may be installed by other users in other locations.